src/math/mod.rs

```diff
@@ -74,14 +87,8 @@ pub use self::round::round;
 pub use self::roundf::roundf;
 pub use self::scalbn::scalbn;
 pub use self::scalbnf::scalbnf;
+pub use self::sin::sin;
 pub use self::sqrt::sqrt;
 pub use self::sqrtf::sqrtf;
 pub use self::trunc::trunc;
 pub use self::truncf::truncf;
-
-mod k_cosf;
-mod k_sinf;
-mod rem_pio2_large;
-mod rem_pio2f;
-
-use self::{k_cosf::k_cosf, k_sinf::k_sinf, rem_pio2_large::rem_pio2_large, rem_pio2f::rem_pio2f};
```

Comment:
These re-export are OK to keep as they are private and won't become part of this crate public API.

Reply:
Okay, I ended up removing that really because it was the single line and I hadn't needed it, I'll change things up so that it's multiple lines.